src/dataLayer/mongo/user-dataLayer.test.js

```diff
@@ -108,3 +110,40 @@ describe('getUser', () => {
     });
   });
 });
+
+describe('deleteUser', () => {
+  it('should delete an existing user', async done => {
+    const result = await createUser({}, {}, validContext);
+    const { accountLinkId } = result;
+    const response = await deleteUser({}, { accountLinkId }, validContext);
+    expect(response).toBeTruthy();
+    expect(response.accountLinkId).toMatch(accountLinkId);
+    const searchResult = await UserModel.findOne({ accountLinkId });
+    expect(searchResult).toBe(null);
+    done();
+  });
+  it('should return with an error message when a user not in db', async done => {
```

Comment:
could you please rephrase so that it does not mention the database? I'd like to future proof for moving some logic to other services. `should return an error  message for a non existing user` would do.